execution-plane/arcon/src/lib.rs

```diff
@@ -34,12 +29,11 @@ pub mod util;
 #[cfg(test)]
 pub mod test_utils {
     use crate::allocator::ArconAllocator;
+    use once_cell::sync::Lazy;
```

Comment:
Any special reason for the swap between lazy_static and once_cell?

Reply:
It's nice IMO. https://docs.rs/once_cell/1.3.1/once_cell/#faq